src/table/internal.tsx

```diff
@@ -349,10 +346,11 @@ const InternalTable = React.forwardRef(
             __internalRootRef={__internalRootRef}
             className={clsx(baseProps.className, styles.root)}
             __funnelSubStepProps={__funnelSubStepProps}
+            __fullPage={variant === 'full-page'}
```

Comment:
It seems like we already pass down the variant: https://github.com/cloudscape-design/components/blob/a247ab2ac7f005f8298e1eede66a991fc53bba81/src/table/internal.tsx#L390

Do we need another one?